src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -2861,11 +2861,11 @@ Right-click to remove the key combination, thereby deactivating the shortcut.</v
   </data>
   <data name="AlwaysOnTop.ModuleDescription" xml:space="preserve">
     <value>Always On Top is a quick and easy way to pin windows on top.</value>
-    <comment>"Always On Top" is the name of the utility</comment>
+    <comment>{Locked="Always On Top"}</comment>
   </data>
   <data name="AlwaysOnTop.ModuleTitle" xml:space="preserve">
     <value>Always On Top </value>
```

Comment:
why is there an extra space here

Reply:
Sorry, where is the extra space?

Reply:
What, it's right here:

<img width="329" alt="image" src="https://github.com/microsoft/PowerToys/assets/189190/9681c131-61c6-410a-8f3c-43a12021ad4f">

Reply:
Sorry, that wasn’t in this PR. Great catch